### DISCUSSION TITLE
Exposing useActions composable to custom components

### DISCUSSION THREAD

**Discussion by andypooletrioteca** [laravel/nova-issues#5906] [Ideas & Feature Requests] [answered]
It would be very useful to have the useActions composable exposed on 'laravel-nova' so that it can be used on custom nova-components. As an example, i'd like to have a custom detail field with a small inline form that ends up triggering a nova Action. To do this i need to replicate much of the logic of the useActions file, as I cannot do `import { useActions } from 'laravel-nova'`. Is there an easy alternative?

**Comment by milewski** [comment, marked as answer]
There is an alternative way, you can first alias the nova resource js directory on your nova.mix.js like this:

```js
webpackConfig.resolve.alias = {
    //...
    '@': path.resolve(__dirname, '../../vendor/laravel/nova/resources/js/'),
}
```

then on your vue components you can import it like this:

```js
import { useActions } from '@/composables/useActions'
```
